### PR TITLE
Nlp

### PR DESCRIPTION
**Pull request by Dave-Gallup**
Set up credentials for Dave to access Google NLP API.  Also corrected small data shape error for client-side ingest.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,10 +23,12 @@
     "eslint-config-airbnb-base": "11.3.1",
     "eslint-plugin-import": "2.7.0",
     "express": "4.15.3",
+    "grpc": "^1.4.1",
```
Comment (joshwyatt) [inline package.json]: @Dave-Gallup please remove the `^` from this dependency we futzed with yesterday so as to lock in its version, thanks!

File: package.json [hunk 2]
```diff
@@ -23,10 +23,12 @@
...
     "humps": "2.0.1",
     "knex": "0.13.0",
     "nodemon": "1.11.0",
     "pg": "6.4.0",
+    "protobufjs": "^6.8.0",
```
Comment (joshwyatt) [inline package.json]: @Dave-Gallup please remove the `^` from this dependency we futzed with yesterday so as to lock in its version, thanks!

File: routes/slack.js [hunk 3]
```diff
@@ -38,19 +38,27 @@ function setEvents(io) {
...
+        newMessage[channelId][messageId].channelId = message[0].channel_map_id;
 
 
         io.sockets.emit('messages', newMessage);
 
+
         analyzeSentimentAndSaveScore(io, message[0].channel_map_id);
       });
-    res.send('SLACK!');
+    res.sendStatus(200);
```
Comment (joshwyatt) [inline routes/slack.js:61]: @Dave-Gallup I accept this change with mild sadness.

**Comment by Dave-Gallup** [comment]
fixed

**Comment by joshwyatt** [comment]
Thanks @Dave-Gallup!!

**Pull request merged**